contract/system/param.go

```diff
@@ -41,15 +41,11 @@ func InitSystemParams(g dataGetter, bpCount int) {
 	systemParams = loadParam(g)
 }
 
-func genParamKey(id string) []byte {
-	return []byte("param\\" + strings.ToUpper(id))
-}
-
 func loadParam(g dataGetter) parameters {
 	ret := map[string]*big.Int{}
 	for i := sysParamIndex(0); i < sysParamMax; i++ {
 		id := i.ID()
-		data, err := g.GetData(genParamKey(id))
+		data, err := g.GetData(schema.KeyParam([]byte(id)))
```

Comment:
What about `SystemParam`?
```go
data, err := g.GetData(dbkey.SystemParam(id))
```
Changes on this file may cause conflict with other open PR

Reply:
ok. if it changed, StakingKey / VoteKey / VoteSortKey, VprKey also changed to SystemXXXKey.

Reply:
It is way better to remove the `Key` from the names and use `dbkey` instead of `schema`
The above would be `dbkey.SystemParam(id)`
If a collision of variable or funcion name happens, just use another name for the them